### ISSUE TITLE
Does not work with raise text card number

### ISSUE THREAD

**Issue by coolraju** [OnixFlutterTeam/ml_card_scanner#16]
The plugin failed to read raised text card numbers. 

![image](https://github.com/user-attachments/assets/f310db43-197c-4ffb-9cd9-3e133d659e5d)

**Comment by gopalmak** [comment]
I'm facing the same issue. Does anyone have any solution?

**Comment by gopalmak** [comment]
Can I also scan the name of the cardholder?

**Comment by cozvtieg9** [comment]
Hi @gopalmak. We just published update `ml_card_scanner: ^0.2.3`. Please try this version.

**Comment by gopalmak** [comment]
Thanks, @cozvtieg9.
Now I'm using `ml_card_scanner: ^0.2.3` and getting below errors :

`Could not resolve org.jetbrains.dokka:dokka-core:1.4.32.
     Required by:
         project :ml_card_scanner > com.android.tools.build:gradle:7.2.0
      > Could not resolve org.jetbrains.dokka:dokka-core:1.4.32.
         > Could not get resource 'https://repo.maven.apache.org/maven2/org/jetbrains/dokka/dokka-core/1.4.32/dokka-core-1.4.32.pom'.
            > Could not GET 'https://repo.maven.apache.org/maven2/org/jetbrains/dokka/dokka-core/1.4.32/dokka-core-1.4.32.pom'.
               > The server may not support the client's requested TLS protocol versions: (TLSv1.2, TLSv1.3). You may need to configure the client to allow other protocols to be used. For more on this, please refer to https://docs.gradle.org/8.10.2/userguide/build_environment.html#sec:gradle_system_properties in the Gradle documentation.
                  > Remote host terminated the handshake`

I'm using 
`id "com.android.application" version "8.7.3" apply false` and `distributionUrl=https\://services.gradle.org/distributions/gradle-8.10.2-all.zip` versions.

**Comment by cozvtieg9** [comment]
@gopalmak Is the issue still reprocudicng? `Remote host terminated the handshake` indicates that there could be connection issues in your network. Is `https://repo.maven.apache.org/maven2/org/jetbrains/dokka/dokka-core/1.4.32/dokka-core-1.4.32.pom` reachable in your browser?

**Comment by gopalmak** [comment]
Yes, @cozvtieg9 `https://repo.maven.apache.org/maven2/org/jetbrains/dokka/dokka-core/1.4.32/dokka-core-1.4.32.pom` is reachable in my browser.

Thanks, Now I can run the app on my actual device.
